pybaseball/statcast_fielding.py

```diff
@@ -42,7 +40,7 @@ def statcast_outfielder_jump(year: int, min_att: Union[int, str] = "q") -> pd.Da
 	return data
 
 @cache.df_cache()
-def statcast_catcher_poptime(year: int, min_2b_att: int, min_3b_att: int) -> pd.DataFrame:
+def statcast_catcher_poptime(year: int, min_2b_att: int = 5, min_3b_att: int = 0) -> pd.DataFrame:
```

Comment:
So now it's not a statcast default?

Reply:
It is what Statcast defaults to, they just don't have a `qualified` option for this leaderboard